misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/README.md

```diff
@@ -1,23 +1,30 @@
 ## Current data mapping
+
 ****
-Values computed in Metasfresh 
+Values computed in Metasfresh
+
 * `Order` - pulled via the search endpoint `api/v3/search/order/`
 
 computed from the following parameters set on window `541116 - ExternalSystem_Config_Shopware6`
-  * `JsonExternalSystemRequest.parameters.JSONPathConstantBPartnerID`
-  * `JsonExternalSystemRequest.parameters.JSONPathConstantBPartnerLocationID`
-  * `JsonExternalSystemRequest.parameters.JSONPathConstantSalesRepID`
-  * `JsonExternalSystemRequest.parameters.ClientId`  
-  * `JsonExternalSystemRequest.parameters.ClientSecret`  
-  * `JsonExternalSystemRequest.parameters.BasePath`
-  * `JsonExternalSystemRequest.parameters.UpdatedAfterOverride` 
-  * `JsonExternalSystemRequest.parameters.UpdatedAfter`
-    
+
+* `JsonExternalSystemRequest.parameters.ClientId`
+* `JsonExternalSystemRequest.parameters.ClientSecret`
+* `JsonExternalSystemRequest.parameters.BasePath`
+* `JsonExternalSystemRequest.parameters.JSONPathConstantBPartnerLocationID`
+* `JsonExternalSystemRequest.parameters.JSONPathEmail`
+* `JsonExternalSystemRequest.parameters.JSONPathConstantSalesRepID`
+* `JsonExternalSystemRequest.parameters.UpdatedAfterOverride`
+* `JsonExternalSystemRequest.parameters.UpdatedAfter`
+* `JsonExternalSystemRequest.parameters.M_FreightCost_NormalVAT_Product_ID`
+* `JsonExternalSystemRequest.parameters.FreightCost_NormalVAT_Rates`
+* `JsonExternalSystemRequest.parameters.M_FreightCost_ReducedVAT_Product_ID`
+* `JsonExternalSystemRequest.parameters.FreightCost_Reduced_VAT_Rates`
+
```

Comment:
because of this empty line all bullets are oddly spaced. if you want do set the following item apart from the others, please find another way..e.g. `-----`

generally, please check the rendered markdown ( https://github.com/metasfresh/metasfresh/blob/59d301bfae8de0e566e5a7d0b867828fba11ee17/misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/README.md ) and make sure it looks OK